sensor_frame: livox_mid360
scan_voxel_size: 0.05
decay_time: 1.0
no_decay_dis: 1.75
clearing_dis: 8.0
use_sorting: True
quantile_z: 0.5
consider_drop: False
limit_ground_lift: False
max_ground_lift: 0.15
clear_dy_obs: True
min_dy_obs_dis: 0.3
min_dy_obs_angle: 0.0
min_dy_obs_rel_z: -0.3
abs_dy_obs_rel_z_thre: 0.2
min_dy_obs_vfov: -28.0
max_dy_obs_vfov: 33.0
min_dy_obs_point_num: 1
no_data_obstacle: False
no_data_block_skip_num: 0
min_block_point_num: 10
vehicle_height: 0.5
voxel_point_update_thre: 100
voxel_time_update_thre: 2.0
min_rel_z: -1.5
max_rel_z: 0.3
dis_ratio_z: 0.2